nipype/algorithms/modelgen.py

```diff
@@ -36,10 +32,11 @@
 
 
 def gcd(a, b):
```

Comment:
Out-of-scope question while we're here: Why are we reimplementing `math.gcd`? (cc @satra)

Reply:
Good question. When was `math.gcd` added? Perhaps this is previous?

Reply:
Opened #3125 3125